static/app/views/projectInstall/issueAlertOptions.spec.tsx

```diff
@@ -31,126 +20,25 @@ describe('IssueAlertOptions', function () {
-      url: `/projects/${organization.slug}/rule-conditions/`,
-      body: MOCK_RESP_VERBOSE,
-    });
-  });
   afterEach(() => {
     MockApiClient.clearMockResponses();
     jest.clearAllMocks();
   });
 
-  it('should render only the `Default Rule` and `Create Later` option on empty response:[]', () => {
-    MockApiClient.addMockResponse({
-      url: URL,
-      body: [],
-    });
-
-    render(getComponent(), {organization});
-    expect(screen.getAllByRole('radio')).toHaveLength(2);
-  });
-
-  it('should render only the `Default Rule` and `Create Later` option on empty response:{}', () => {
-    MockApiClient.addMockResponse({
-      url: URL,
-      body: {},
-    });
-
-    render(getComponent(), {organization});
-    expect(screen.getAllByRole('radio')).toHaveLength(2);
-  });
-
-  it('should render only the `Default Rule` and `Create Later` option on responses with different allowable intervals', () => {
-    MockApiClient.addMockResponse({
-      url: URL,
-      body: MOCK_RESP_INCONSISTENT_INTERVALS,
-    });
-
-    render(getComponent(), {organization});
-    expect(screen.getAllByRole('radio')).toHaveLength(2);
-  });
-
-  it('should render all(three) options on responses with different placeholder values', () => {
-    MockApiClient.addMockResponse({
-      url: URL,
-      body: MOCK_RESP_INCONSISTENT_PLACEHOLDERS,
-    });
-    render(getComponent(), {organization});
-    expect(screen.getAllByRole('radio')).toHaveLength(3);
-  });
-
-  it('should ignore conditions that are not `sentry.rules.conditions.event_frequency.EventFrequencyCondition` and `sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition`', async () => {
-    MockApiClient.addMockResponse({
-      url: URL,
-      body: MOCK_RESP_ONLY_IGNORED_CONDITIONS_INVALID,
-    });
-
-    render(getComponent(), {organization});
-    expect(screen.getAllByRole('radio')).toHaveLength(3);
-    await selectEvent.select(screen.getByText('Select...'), 'users affected by');
-    expect(props.onChange).toHaveBeenCalledWith(
-      expect.objectContaining({
-        defaultRules: false,
-        shouldCreateCustomRule: true,
-      })
-    );
-  });
-
-  it('should render all(three) options on a valid response', () => {
-    MockApiClient.addMockResponse({
-      url: URL,
-      body: MOCK_RESP_VERBOSE,
-    });
-
-    render(getComponent());
-    expect(screen.getAllByRole('radio')).toHaveLength(3);
-  });
-
-  it('should pre-populate fields from server response', async () => {
-    MockApiClient.addMockResponse({
-      url: URL,
-      body: MOCK_RESP_VERBOSE,
-    });
-
-    render(getComponent());
-    await selectEvent.select(screen.getByText('occurrences of'), 'users affected by');
-    await selectEvent.select(screen.getByText('one minute'), '30 days');
-    expect(props.onChange).toHaveBeenCalledWith(
-      expect.objectContaining({
-        defaultRules: false,
-        shouldCreateCustomRule: true,
-      })
-    );
-  });
```

Comment:
These tests were checking for different cases when the network request returned different data. None of that matters anymore. Also the names of the tests were good, but the assertions inside didn't always match... for example "should render all(three) options on responses with different placeholder values" did not assert placeholder values were set.